.circleci/config.yml

```diff
@@ -306,7 +309,7 @@ jobs:
     docker:
       - image: circleci/node
     environment:
-      TERM: xterm
+      TERM: << pipeline.parameters.environment-term >>
```

Comment:
What I meant was to have the variable automatically declared for all jobs.

Maybe that's not possible - part of the task is to check that. Looking at [Environment variable usage options > Order of precedence](https://circleci.com/docs/2.0/env-vars/#order-of-precedence), the highest level that is still possible via `.config.yml` file be the `job` dict, which we are already doing.

If it's not possible, then I'd just leave the variable is it was because doing it via parameters is actually more verbose and still requires you to remember to put it on a job.